### PR TITLE
les: polish code

### PR DESCRIPTION
**Pull request by rjl493456442**
This PR polishes the les internal a bit. Nothing important changed.

### CODE REVIEW

File: les/vflux/server/prioritypool.go [hunk 1]
```diff
@@ -461,14 +470,12 @@ func (pp *priorityPool) updateFlags(updates []capUpdate) {
...
 		c := pp.inactiveQueue.PopItem().(*ppNodeInfo)
-		pp.setTempState(c)
+		pp.setTempState(c, pp.minCap, pp.capacityStepDiv, pp.activeBias)
 		pp.setTempCapacity(c, pp.minCap)
-		c.bias = pp.activeBias
 		pp.activeQueue.Push(c)
 		pp.enforceLimits()
 		if c.tempCapacity > 0 {
 			commit = true
-			c.bias = 0
```
Comment (zsfelfoldi) [inline les/vflux/server/prioritypool.go:471]: This is not an identical change here and I am not sure about the potential effects. Until now, the bias was always only applied to one node that has been inserted or whose capacity was increased. It is meant to protect against quick oscillations so we always use it against the node that is trying to push out others from the pool. Once we decide to grant it the space it needs, it's no longer biased. Now `tryActivate` can activate multiple clients and in your version the previously activated ones stay biased until we finish the entire operation. This means that further inactive nodes can very easily deactivate recently activated ones if their priorities are very close to each other because they are both biased and the already activated one does not have an advantage against the latest activation attempt. I'd prefer to keep the old and thoroughly tested behavior here.

File: les/vflux/server/prioritypool.go [hunk 2]
```diff
@@ -311,40 +307,55 @@ func (pp *priorityPool) connectedNode(c *ppNodeInfo) {
...
+	pp.lock.Unlock()
+	pp.updateFlags(updates)
 }
 
 // setTempState internally puts a node in a temporary state that can either be reverted
 // or confirmed later. This temporary state allows changing the capacity of a node and
 // moving it between the active and inactive queue. activeFlag/inactiveFlag and
 // capacityField are not changed while the changes are still temporary.
-func (pp *priorityPool) setTempState(c *ppNodeInfo) {
+func (pp *priorityPool) setTempState(c *ppNodeInfo, minTarget uint64, stepDiv uint64, bias time.Duration) {
```
Comment (zsfelfoldi) [inline les/vflux/server/prioritypool.go:332]: It's a nice thing that temp parameter changes are explicitly tied to the temp state but I'm not sure that they really need to be set in the same function that enters the temp state. In many cases they are not changed at all, and in `tryActivate` I think they might need to change again (set bias to zero) while still in temp state (see my comment there). So maybe it'd be better to have a separate function for entering the temp state and another one for changing the temp state parameters (which could just do a check that we are really in temp state). Just like `setTempCapacity` that can change `tempCapacity` any time while in temp state.

File: les/server_handler.go [hunk 3]
```diff
@@ -151,19 +153,14 @@ func (h *serverHandler) handle(p *clientPeer) error {
 		p.Log().Debug("Client pool already closed")
 		return p2p.DiscRequested
 	}
-	activeCount, _ := h.server.clientPool.Active()
-	clientConnectionGauge.Update(int64(activeCount))
```
Comment (zsfelfoldi) [inline les/server_handler.go:155]: Why did you remove these updates? Do you think this metric is useless? Btw the metric itself is still there is `metrics.go` so this looks wrong.

**Review by zsfelfoldi — CHANGES_REQUESTED** [review]

File: les/server_handler.go (hunk 3 above)
Comment (rjl493456442) [reply les/server_handler.go:155]: Because they are already counted in the clientpool internal. https://github.com/ethereum/go-ethereum/blob/master/les/vflux/server/clientpool.go#L154

File: les/vflux/server/prioritypool.go (hunk 1 above)
Comment (rjl493456442) [reply les/vflux/server/prioritypool.go:471]: Gotcha. It's my mistake, will fix it.

**Comment by rjl493456442** [comment]
@zsfelfoldi I think all the issues are resolved, please take another look!

**Review by zsfelfoldi — APPROVED** [review]
If `clientConnectionGauge` is not updated in package `les` any more then we should remove it. Otherwise LGTM.
